parler_rest/fields.py

```diff
@@ -3,6 +3,8 @@
 Custom serializer fields for nested translations.
 """
 from __future__ import unicode_literals
+import json
+from past.builtins import basestring
```

Comment:
Thanks @vdboor, does it work for detecting an instance of string (str or utf8) with python 2 and python 3?